enumeration/extension/enum-High.yaml

```
@@ -0,0 +1,24 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: enumeration
+
+uri: https://github.com/glamberson/gedcom-evidence/enum-High
+
+extension tags: [ _HIGH ]
```

```suggestion
extension tags:
  - _HIGH
```

Luther commented on my PR:
> The [format description's style guide](https://gedcom.io/terms/format#readable-yaml) says
>
> [Sequences](https://yaml.org/spec/1.2.2/#821-block-sequences) and [mappings](https://yaml.org/spec/1.2.2/#822-block-mappings) should be in the block style unless they are empty, in which case a [flow style](https://yaml.org/spec/1.2.2/#74-flow-collection-styles) should be used instead.

>
> This same change would apply in many other places in this PR